docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/amazon-eks.md

```diff
@@ -31,6 +31,43 @@ that uses the Amazon EBS `gp3` volume type. Then, use it cluster-wide as a defau
 If you encounter issues with EBS CSI Driver, follow the instructions in the [helm-profiles repository](https://github.com/camunda-community-hub/camunda-8-helm-profiles/blob/main/aws/README.md#ebs-csi-driver-addon) maintained by the Camunda Consulting Team.
 :::
 
+## Load Balancer Setup
+
+AWS is offering different types of Load Balancers (LB). Those namely being Classic Load Balancer (CLB), Network Load Balancer (NLB), and Application Load Balancer (ALB). Typically the NLB and ALB are used in production setups.
```

Comment:
I'd make this a bullet point list so you can also directly mention that one option is deprecated:

* CLB (deprecated, don't use)
* ALB
* NLB

Reply:
Just a side note, I'm not sure CLB are deprecated, at least couldn't find anything about it anymore.
I think it was a misconception that somehow persisted with us. AWS discontinued the EC2 Classic Networking and the Classic LB connected with it but Classic LBs itself still work and can be created without an official deprecation announcement. They essentially discontinued any pre VPC usage.

Nevertheless, I agree with the bullet point list!

Reply:
there's nothing that officially says `deprecated` but at least when creating one it's called `previous generation` and kinda hidden, so not really endorsed anymore.